server/resources/translations/common.es_us.json

```diff
@@ -1,6 +1,7 @@
 {
   "alertAcknowledged": "Otro personal de auxilio ha confirmado esta solicitud. (No necesita responder a este mensaje).",
   "alertAdditionalAlert": "Se generó una alerta {{ alertTypeDisplayName }} adicional en {{ deviceDisplayName }}",
+  "alertAdditionalAlertWithResetChoice": "",
```

Comment:
For the strings that Ahmad didn't have a translation for yet, he used Google Translate. I think that's a good first estimate on these that we allow us to deploy the code without waiting on our translators and that you can do that for these strings too.

These are the official translations that we have right now for Spanish: https://docs.google.com/spreadsheets/d/1sTUUpZFH4c4isM_pq07xNBjZvoMUtm-B/edit?usp=drive_web&ouid=116668426689675846861&rtpof=true

You can add the new strings that you need to that and to this template spreadsheet that we plan to use for any future languages: https://docs.google.com/spreadsheets/d/10KrtQEo-LQPoHxX_ZJdZEp-U3D6yAuqHEkXvJun1rAg/edit#gid=0

Can you please also double-check that all of Ahmad's new strings are in there too? Some of them were talked about here: https://bravely.slack.com/archives/GD0HUUML3/p1698336165531649

And then can you please get a hold of Oona to let her know to contact her Spanish translators to get these strings officially translated? This won't hold up deployment, but needs to be done eventually. I also highly recommend that you add a reminder to yourself to keep poking Oona about this every week or two because it probably won't be high on her list otherwise.

Thanks!

Reply:
Done this!

Note for the future; we need Spanish translations for:

- "alertAdditionalAlertWithResetChoice": "An additional {{ alertTypeDisplayName }} alert was generated at {{ deviceDisplayName }}.\n\nWe noticed that many alerts are being sent from {{ deviceDisplayName }}. Please respond with 'ok' once you have checked on it, or 'reset' to reset the device if you believe it is malfunctioning."
- "resetNoticeToOtherResponders": "Another Responder has acknowledged this request and decided to reset the Brave Sensor. (You don't need to respond to this message.)"
- "resetRequestAccepted": "This Brave Sensor will be reset, and you will no longer receive alerts for this session.\n\nPlease note that after a reset, the sensor will not accurately monitor for stillness until about ten minutes have passed. If you have not checked on this location yet, please do so.\n\nThis session is now complete. (You don't need to respond to this message.)"
- "resetRequestClientMessage": "reset"
- "resetRequestDeclined": "Sorry, you can only reset this Brave Sensor if it is sending an abnormal number of alerts. We will let you know when this becomes an option.\n\nPlease respond with 'ok' once you have checked on it."